samples/AzureFunctionsApp/AzureFunctionsApp.cs

```diff
@@ -15,8 +15,6 @@ public static class AzureFunctionsApp
         public static async Task<List<string>> RunOrchestrator(
             [OrchestrationTrigger] TaskOrchestrationContext context)
         {
-            ILogger logger = context.CreateReplaySafeLogger(nameof(AzureFunctionsApp));
-            logger.LogInformation("Saying hello.");
```

Comment:
I'll add this back right after the change to make `CreateReplaySafeLogger` virtual is released.

Reply:
Will this be needed if you use `Moq.Protected`?

Reply:
Nope, added it back just now